zigpy/device.py

```diff
@@ -338,13 +338,26 @@ def __getitem__(self, key):
         return self.endpoints[key]
 
     def get_signature(self):
-        signature = {}
+        # return the device signature by providing essential device information
+        #    - Model Identifier ( Attribut 0x0005 of Basic Cluster 0x0000 )
+        #    - Manufacturer Name ( Attribut 0x0004 of Basic Cluster 0x0000 )
+        #    - Endpoint list
+        #        - Profile Id, Device Id, Cluster Out, Cluster In
+        signature = {"manufacturer_name": '', "manufacture_code": '', "model": ''}
```

Comment:
Just keep it empty.

Reply:
what do you mean ? I should keep signature empty, and then initialize each element ?

Reply:
yes. do `signature = {}` if model, manufacturer are `None` then they should be missing from the sig